toast-tk-runtime/src/main/java/io/toast/tk/runtime/result/ObjectResultHandler.java

```diff
@@ -9,10 +14,39 @@
     public ObjectResultHandler(IActionItemRepository objectRepository) {
         super(objectRepository);
     }
-    
+
+	@SuppressWarnings({ "unchecked", "rawtypes" })
+    private String resultBuetifier(Object value) {	
+		try {
+			return ((List)value).stream()
+					.map(Object::toString)
+					.collect(Collectors.joining(System.lineSeparator()))
+					.toString();
+		}
+		catch(Exception e) {
+			try {
+		    	StringBuilder res = new StringBuilder();
+				for(Entry entry : ((Map<?, ?>) value).entrySet()) {
+					res.append(entry.getKey().toString() + " = " 
+							+ entry.getValue().toString() 
+							+ System.lineSeparator());
+				}
+				return res.toString();
+			} catch(Exception e2) {
+				if(value != null) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2583)